cogs/quotes.py

```diff
@@ -199,9 +199,13 @@ def check(reaction, user):
         # current nick will be displayed, otherwise use the name stored
         # in db
 
-        author_name_unesc = author.display_name if author else name
```

Comment:
I think doing author.mention could be better

Reply:
oop i selected the wrong line, but it's the same for the new one

Reply:
So `author_name = author.mention if author else name`

Reply:
We have discovered it is not possible to mention users in author_name, only the ID appears